base/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM phusion/passenger-ruby26:1.0.10
+
+RUN echo 'Downloading Packages' && \
+  curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
+  curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+  echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
+  apt-get update -qq && \
+  apt-get install -y --no-install-recommends \
+    build-essential \
+    libsasl2-dev \
+    nodejs \
+    rsync \
+    tzdata \
+    yarn \
+  && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
+  echo 'Packages Downloaded'
+
+# Install Chrome so we can run system specs for Blacklight
```

Comment:
Do we want to keep this, or do we want to set up Chrome in the docker-compose / .circleci/config.yml?

Reply:
I think yes, but that is separate scope and not part of this ticket. Both projects will likely need feature specs (as might Camerata) which means Chrome, but having it spin up separate in docker-compose works nicely. We're set to meet about the various compose level parts this week and I would assume a ticket about this would come out of that meeting.